terraform/modules/eks/main.tf

```diff
@@ -0,0 +1,169 @@
+data "aws_eks_cluster" "cluster" {
+  name = module.eks.cluster_id
+}
+
+data "aws_eks_cluster_auth" "cluster" {
+  name = module.eks.cluster_id
+}
+
+provider "kubernetes" {
+  host                   = data.aws_eks_cluster.cluster.endpoint
+  cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)
+  token                  = data.aws_eks_cluster_auth.cluster.token
+}
+
+module "eks" {
+  source          = "terraform-aws-modules/eks/aws"
+  version         = "18.10.1"
+  cluster_name    = var.cluster_name
+  cluster_version = "1.21"
+
+  subnet_ids = var.subnet_ids
+  vpc_id     = var.vpc_id
+
+  self_managed_node_group_defaults = {
+    instance_type                = "t2.small"
+  }
+
+  self_managed_node_groups = {
+    worker-group-1 = {
+      instance_type                 = "t2.small"
+      additional_security_group_ids = var.worker_sg_ids
+      min_size                      = 3
+      max_size                      = 8
+      desired_size                  = 3
+    }
+  }
+
+  # Extend cluster security group rules
+  cluster_security_group_additional_rules = {
```

Comment:
Same here. As i can see in eks module is defined as `List of additional security group ` just like we did in the other modules

Reply:
This is just a cluster specific security group and I think it makes sense to define it in place. The same approach is used in the terraform examples